sentry-contrib-native-sys/src/lib.rs

```diff
@@ -257,6 +257,9 @@ extern "C" {
     #[link_name = "sentry_event_value_add_stacktrace"]
     pub fn event_value_add_stacktrace(event: Value, ips: *mut *mut c_void, len: usize);
 
+    #[link_name = "sentry_value_add_stacktrace"]
```

Comment:
I will comment on the sentry-native issue momentarily with an idea that I have.

Reply:
Had to remove it in https://github.com/daxpedda/sentry-contrib-native/commit/1bbf83ed7b561fad82d05eb07bbdcf71c0f83589, sorry @Jake-Shadle!

Reply:
> ... if you prefer, I could reimplement it on the Rust side ...

That sounds good too!